lib/route.js

```diff
@@ -82,7 +82,9 @@ function buildRouting (options) {
       return503OnClosing = Object.prototype.hasOwnProperty.call(options, 'return503OnClosing') ? options.return503OnClosing : true
       buildPerformanceValidator = ValidatorSelector()
     },
-    routing: router.lookup.bind(router), // router func to find the right handler to call
+    routing: (...args) => {
```

Comment:
Unless we need an arrow function, I'd rather see `routing function () {}` or `routing () {}`.

Reply:
Sorry yep this was just for debugging